flutter/test/screenshot/recorder_config_test.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:sentry_flutter/src/screenshot/recorder_config.dart';
+
+void main() async {
+  group('$ScreenshotRecorderConfig', () {
+    test('defaults', () {
+      var sut = ScreenshotRecorderConfig();
+      expect(sut.height, isNull);
+      expect(sut.width, isNull);
```

Comment:
```suggestion
      expect(sut.width, isNull);
      expect(sut.getPixelRatio(100, 100), isNull);
```

Reply:
I have the line `expect(ScreenshotRecorderConfig().getPixelRatio(100, 100), null);` already in line 14. I don't think having it twice would make sense.